Update guidance for length of time files are available (#201)
alphagov/notifications-tech-docs: source/documentation/_api_docs.md

Set the number of weeks you want the file to be available using the `retention_period` key.

> You can choose any value between 1 week and 78 weeks. When deciding this, you should consider:
> 
> * the risk to the recipient if the file contains personal information
> * whether they’ll need to download the file again later

Author: ```suggestion
You can choose any value between 1 week and 78 weeks. When deciding this, you should consider:

* the need to protect the recipient’s personal information
* whether the recipient will need to download the file again later
```
